sections/transactions.js

```diff
@@ -263,4 +263,22 @@ export default [
     type: "text",
     content: "You can check the property `knex.isTransaction` to see if the current knex instance you are working with is a transaction."
   },
+  {
+    type: "text",
+    content: "In case you need to specify an isolation level for your transaction, you can use a config parameter `isolationLevel`. Not supported by oracle and sqlite, options are read uncommitted, read committed, repeatable read, snapshot (mssql only), serializable."
```

Comment:
```suggestion
    content: "In case you need to specify an isolation level for your transaction, you can use a config parameter `isolationLevel`. Not supported by oracle and sqlite, options are `read uncommitted`, `read committed`, `repeatable read`, `snapshot` (mssql only), `serializable`."
```